packages/tokens/package.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "@openfun/cunningham-tokens",
+  "private": false,
+  "version": "0.0.0",
+  "license": "MIT",
+  "bin": {
+    "cunningham": "dist/bin/Main.js"
+  },
+  "exports": {
+    "./default-tokens": "./dist/cunningham-tokens.css"
+  },
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "lint": "eslint . '**/*.ts?(x)'",
```

Not sure that currently the linter checks ts files. I believe you are currently testing files containing `.tsx` or `.tx`.

```suggestion
    "lint": "eslint . '**/*.(js|ts)x?$'",
```